modules/react/tooltip/lib/TooltipContainer.tsx

```diff
@@ -86,16 +86,16 @@ export const TooltipContainer = styled('div')<TooltipContainerProps>(
 
     // Fix offsets based on placement
     '[data-popper-placement="top-start"] &, [data-popper-placement="bottom-start"] &': {
-      left: -spaceNumbers.xxxs,
+      left: `${-space.xxxs}`,
```

Comment:
```suggestion
      left: `-${space.xxxs}`,
```

Should it be this way? Because `space.xxxs` is a string, js can convert it to negative number and remove `rem` part from it.

Reply:
This has been updated